deploy/setup-project.sh

```diff
@@ -22,34 +22,65 @@ readonly KUBEDEPLOY="${PKGDIR}/deploy/kubernetes"
 readonly BIND_ROLES="roles/compute.storageAdmin roles/iam.serviceAccountUser projects/${PROJECT}/roles/gcp_compute_persistent_disk_csi_driver_custom_role"
 readonly IAM_NAME="${GCE_PD_SA_NAME}@${PROJECT}.iam.gserviceaccount.com"
 
-# Create or Update Custom Role
-if gcloud iam roles describe gcp_compute_persistent_disk_csi_driver_custom_role --project "${PROJECT}";
-then
-  gcloud iam roles update gcp_compute_persistent_disk_csi_driver_custom_role --quiet \
-          --project "${PROJECT}"                                                     \
-          --file "${PKGDIR}/deploy/gcp-compute-persistent-disk-csi-driver-custom-role.yaml"
-else
-  gcloud iam roles create gcp_compute_persistent_disk_csi_driver_custom_role --quiet \
-    --project "${PROJECT}"                                                           \
-    --file "${PKGDIR}/deploy/gcp-compute-persistent-disk-csi-driver-custom-role.yaml"
+function create_new_sa()
```

Comment:
move function definitions to bottom of file

Reply:
Undid the extraction to a function as bash needs functions before usage, but it's not so bad inline